01-02_Loops/Homework/OkoBere.py

```diff
@@ -5,4 +5,46 @@
 # když přesně => vypíšu že vyhrál
 # když víc => vypíšu o kolik a že prohrál
 
-#(while + random)
\ No newline at end of file
+#(while + random)
+
+from random import randrange
+
+def main():
+    cislo_kasino()
+    cislo_uzivatel()
+    vypocet()
+
+def cislo_kasino():
+    return randrange(2, 10)
+
+def cislo_uzivatel():
+    return randrange(2, 10)
+
+def vypocet():
```

Comment:
funkce vypocet() dělá celou logiku i když by měla dělat jen výpočet

Reply:
Přejmenováno, tudíž nemusí dělat už jen výpočet :D

Reply:
Budeme dělat, že jsem tuhle změnu neviděl a společně to předěláme :D